### ISSUE TITLE
very slow app startup

### ISSUE THREAD

**Issue by sherpya** [apigee-127/sway#86]
I'm currently using swagger-restify-mw 0.1.0 that does not use sway, while updating to 0.6.0,
I've noticed the "new world" is to use sway 0.6.0 (1.0.0 it's not yet compatible), but app startup is very slow:

without sway:
```
time node app.js 

real    0m2.200s
user    0m2.284s
sys     0m0.080s
```

with sway:
```
time node app.js

real    0m33.319s
user    0m33.444s
sys     0m0.104s
```

It also fails from swaggerUI with this error:
```
  pipes caught error: Error: req.query must be provided for 'query' parameters
```

in

```
node_modules/swagger-restify-mw/node_modules/swagger-node-runner/node_modules/sway/lib/types/parameter.js:177:13
```

there is something changed that I should be aware of?

**Comment by sherpya** [comment]
I've "cheated", looks like 1.0.0 does not have slow startup problem, but req.query still persist

**Comment by sherpya** [comment]
I've moved express compat before validator, but I don't have anymore `req.swagger.params` in my user code

**Comment by whitlockjc** [comment]
You might have to get `swagger-node-runner` to fix the restify support because it basically drives the Sway API.  As for slow startup, there is a known issue with Sway caused by `json-schema-faker` but if you're not seeing it, maybe they fixed this upstream.  But it was never more than 1s of startup time.

**Comment by sherpya** [comment]
I've managed to fix compatibly problems with `swagger-node-runner` but it's out of this project, the slow startup is fixed when switch from sway 0.6.0 to 1.0.0

**Comment by whitlockjc** [comment]
Good to hear.  Let me know if I can make Sway better.
